.github/workflows/build-test-windows.yml

```
@@ -45,6 +45,13 @@ jobs:
       - name: Build Triton
         run: |
           cd ${{ env.NEW_WORKSPACE }}
+
+          cmd /c '"C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Auxiliary\Build\vcvarsall.bat" x64 && set' | ForEach-Object {
+              if ($_ -match '^(.*?)=(.*)$') {
+                  [Environment]::SetEnvironmentVariable($matches[1], $matches[2])
+              }
+          }
```

This should also work (and simpler).
```suggestion
          Invoke-BatchFile "C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Auxiliary\Build\vcvarsall.bat"
```

---

Invoke-BatchFile is a non-standard extension which is not installed by default. I actually like that Anatoly found a solution that doesn't require installing it, we may want to use it instead. Also save environment variables into GITHUB_ENV for further steps.

---

I've rolled back the last changes and am going to merge the version that was approved. Feel free to ask me to finish something, I can do it in a separate pull request.